rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -46,6 +49,7 @@ std::string format_storage_uri(const std::string & base_folder, uint64_t storage
 }
 }  // namespace
 
+// TODO(piraka9011) Initialize defaults in header file instead.
```

Comment:
Also add a TODO for replacing old-style constructors with uniform initialization